python_bindings/tutorial/CMakeLists.txt

```diff
@@ -16,52 +16,64 @@ set(tests
     lesson_14_types.py
     )
 
+set(PYPATH_lesson_10_aot_compilation_run "$<TARGET_FILE_DIR:lesson_10_halide>")
+
 foreach (test IN LISTS tests)
+    if (TARGET_WEBASSEMBLY AND Halide_TARGET MATCHES "wasm" AND test MATCHES "lesson_10")
+        message(WARNING "Not all tutorials build under WASM.")
+        continue()
+    endif ()
+
+    cmake_path(GET test STEM test_name)
     add_python_test(
         FILE "${test}"
         LABEL python_tutorial
-        PYTHONPATH "$<TARGET_FILE_DIR:lesson_10_halide>"
+        PYTHONPATH "${PYPATH_${test_name}}"
     )
 endforeach ()
 
-## Add some hacks for getting CMake to delay compiling lesson_10_halide until after the test has run. The "better" way
-## of doing this might be to treat lesson 10 like an app and give it its own CMakeLists.txt, but since this is a one-off
-## it is probably less maintenance work to do it like this.
+if (TARGET_WEBASSEMBLY AND Halide_TARGET MATCHES "wasm")
+    message(WARNING "Not all tutorials build under WASM.")
```

Comment:
return()

Reply:
If I do that, the packaging stuff below won't be processed

Reply:
(This particular tutorial really just needs to be rewritten to be way less janky; using PyGen when it lands is arguably the right thing, but for now, yuck)

Reply:
ahhh GitHub's interface was confusing me. I thought this was the end of the file.